aos/closed_loop/calcZernikesFromDonutStamps.ipynb

```diff
@@ -8,10 +8,10 @@
     "# CalcZernikes from DonutStamps\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    plt.ylabel('Zernike Coefficient (microns)')
</pre><p>I'd add something to display only integers given that we don't use fractional Zk polynomials, eg. </p><pre class="ql-syntax" spellcheck="false">from matplotlib.ticker import MaxNLocator&nbsp;
ax.xaxis.set_major_locator(MaxNLocator(integer=True))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Nice, that does look better.</p>